### PR TITLE
Fix widevine loading failure due to different TeamID

### PR DESCRIPTION
**Pull request by simonhong**
Set disable-library-validation entitlements for helper-app that
loads widevine library.

fix https://github.com/brave/brave-browser/issues/5433
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: app/helper-entitlements.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.cs.disable-library-validation</key>
```
Comment (simonhong) [inline app/helper-entitlements.plist]: `helper-plugin-entitlements.plist` has one more property(`allow-unsigned-executable-memory`).
Not sure `helper` needs that also.

Comment (mkarolin) [reply app/helper-entitlements.plist]: According to the comment here https://chromium.googlesource.com/chromium/src/+/3fa9da4e877a, `com.apple.security.cs.allow-unsigned-executable-memory required to
  run Flash content and NaCl modules`. Not sure if widevine needs the same, but the approach with adding the other entitlement makes sense.

Comment (simonhong) [reply app/helper-entitlements.plist]: @mkarolin Thanks for checking.
I think we don't need to add `allow-unsigned-executable-memory` for `helper`.
If this entitlement should be used for widevine, upstream also needs it.

**Comment by bsclifton** [comment]
Ready for uplifts back to 0.68.x 😄

**Pull request merged**